### DISCUSSION TITLE
Any plans for open integration/documentation of internal analytic metadata props?

### DISCUSSION THREAD

**Discussion by wysez** [cloudscape-design/components#2784] [Q&A] [answered]
Hi CSD Team!

Currently leveraging CSD in an internal setting and was super interested in the analytic metadata (albeit internal to AWS) that has been steadily introduced to CSD's components. I understand that that `analyticMetadata` prop is internal to AWS-- but why is that? Does that mean there is no possible way for an outside user like me to leverage this functionality, or is that just something the cloudscape team will not release documentation on? 

 Thanks so much in advance for your time!

**Comment by johannes-weber** [comment, marked as answer]
Hi @wysez, the `analyticMetadata` property is something we experiment with. We do not recommend to make use of it because it's not part of the public API and it may change in the future.
